crates/eframe/src/web/web_painter_wgpu.rs

```diff
@@ -111,6 +112,7 @@ impl WebPainterWgpu {
             present_mode: options.wgpu_options.present_mode,
             alpha_mode: wgpu::CompositeAlphaMode::Auto,
             view_formats: vec![render_state.target_format],
+            desired_maximum_frame_latency: 2, // TODO(emilk): expose to eframe users
```

Comment:
nope

Reply:
very much approving that you keep to the default of 2 because dealing with the fallout of 1 is pretty much an expert feature right now.
It would aactually be even better if we were to use [`..surface.get_default_config`](https://docs.rs/wgpu/latest/wgpu/struct.Surface.html#method.get_default_config) here :)


(oh and by the way I broke DX12 for a random assortment of drivers with the changes that were made to that backend when introducing the feature https://github.com/gfx-rs/wgpu/issues/5088)